From 086d9cef7351c2a3c9322a904ae8b38be94eee34 Mon Sep 17 00:00:00 2001
From: Ludovic Desroches <ludovic.desroches@atmel.com>
Date: Mon, 29 Oct 2012 15:32:54 +0100
Subject: [PATCH 124/319] video: atmel_hlcdfb: increase the base layer AHB
 burst length

Increase the base layer AHB burst length from 4 to 16.

Signed-off-by: Ludovic Desroches <ludovic.desroches@atmel.com>
---
 drivers/video/atmel_hlcdfb.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/video/atmel_hlcdfb.c b/drivers/video/atmel_hlcdfb.c
index 7c87f88..1ae48da 100644
--- a/drivers/video/atmel_hlcdfb.c
+++ b/drivers/video/atmel_hlcdfb.c
@@ -317,7 +317,7 @@ static int atmel_hlcdfb_setup_core_base(struct fb_info *info)
 	dev_dbg(info->device, "  * LCDC_LCDCFG4 = %08lx\n", value);
 	lcdc_writel(sinfo, ATMEL_LCDC_LCDCFG4, value);
 
-	lcdc_writel(sinfo, ATMEL_LCDC_BASECFG0, LCDC_BASECFG0_BLEN_AHB_INCR4 | LCDC_BASECFG0_DLBO);
+	lcdc_writel(sinfo, ATMEL_LCDC_BASECFG0, LCDC_BASECFG0_BLEN_AHB_INCR16 | LCDC_BASECFG0_DLBO);
 	lcdc_writel(sinfo, ATMEL_LCDC_BASECFG1, atmel_hlcdfb_get_rgbmode(info));
 	lcdc_writel(sinfo, ATMEL_LCDC_BASECFG2, 0);
 	lcdc_writel(sinfo, ATMEL_LCDC_BASECFG3, 0);	/* Default color */
@@ -357,7 +357,7 @@ static int atmel_hlcdfb_setup_core_ovl(struct fb_info *info)
 	}
 
 	lcdc_writel(sinfo, ATMEL_LCDC_OVRCFG0,
-			LCDC_OVRCFG0_BLEN_AHB_INCR4 | LCDC_OVRCFG0_DLBO);
+			LCDC_OVRCFG0_BLEN_AHB_INCR16 | LCDC_OVRCFG0_DLBO);
 	lcdc_writel(sinfo, ATMEL_LCDC_OVRCFG1,
 			atmel_hlcdfb_get_rgbmode(info));
 	lcdc_writel(sinfo, ATMEL_LCDC_OVRCFG2, xpos |
-- 
1.7.10.4

